src/extension_constants.h

```diff
@@ -10,6 +10,7 @@
  * loader*/
 
 #define EXTENSION_NAME "timescaledb"
+#define EXTENSION_FDW_NAME "timescaledb_fdw"
 #define TSL_LIBRARY_NAME "timescaledb-tsl"
```

Comment:
Do we still need this separation in the future?

Reply:
Yes, due to license differences.